File: cloud/dao/pay_wallet_recharge_package.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// PayWalletRechargePackage 充值套餐表 pay_wallet_recharge_package
type PayWalletRechargePackage struct {
	Id         *int64        `db:"id" json:"id"`                  //bigint 编号,PRI
	Name       *string       `db:"name" json:"name"`              //varchar 套餐名称
	PayPrice   *int64        `db:"pay_price" json:"payPrice"`     //bigint 支付金额
	BonusPrice *int64        `db:"bonus_price" json:"bonusPrice"` //bigint 赠送金额
	Status     *int32        `db:"status" json:"status"`          //tinyint 状态
	Deleted    *int32        `db:"deleted" json:"deleted"`        //tinyint 删除
	TenantId   *int64        `db:"tenant_id" json:"tenantId"`     //bigint 租户
	Creator    null.String   `db:"creator" json:"creator"`        //varchar 创建人
	CreateTime null.DateTime `db:"create_time" json:"createTime"` //datetime 创建时间
	Updater    null.String   `db:"updater" json:"updater"`        //varchar 更新人
	UpdateTime null.DateTime `db:"update_time" json:"updateTime"` //datetime 更新时间
}

File: admin/src/api/interface/payWalletRechargePackage.ts
// pay_wallet_recharge_package 充值套餐表
import { ReqPage } from "./index";
export namespace PayWalletRechargePackage {
  export interface ReqPayWalletRechargePackageList extends ReqPage {
    deleted?: number; // 删除
    status?: number; // 状态
    name?: string; // 套餐名称
  }
  export interface ResPayWalletRechargePackageItem {
    id: number; // 编号
    name: string; // 套餐名称
    payPrice: number; // 支付金额
    bonusPrice: number; // 赠送金额
    status: number; // 状态
    deleted: number; // 删除
    tenantId: number; // 租户
    creator: string | undefined; // 创建人
    createTime: string | undefined; // 创建时间
    updater: string | undefined; // 更新人
    updateTime: string | undefined; // 更新时间
  }
}
